2.0.1/CHANGELOG.md

```diff
@@ -3,6 +3,6 @@
 Astronomer Certified 2.0.1-1 2020-01-21
 -----------------------------------------
 
+## Bugfixes
 
-
-
+- Bugfixes Done
```

Comment:
This is not a useful changelog entry.

Reply:
I need to work with Kaxil on this as how I can get the exact changelog. Currently I am trying to make sure that I can generate a 2.0.1 build.

Reply:
@ernest-kr I had already linked it to you on how we do it :) in the last PR

https://github.com/astronomer/ap-airflow/pull/200#discussion_r562603325

>We don't add the entire Airflow changelog here, we only add what is new compared to Airflow 2.0.1

Example: https://github.com/astronomer/ap-airflow/blob/master/2.0.0/CHANGELOG.md#astronomer-certified-200-1-2020-12-17

Reply:
yes that part I did not get it. So probably need to understand it clearly. When we are upto date with 2.0.1. I am not sure what else could be added to the changelog. Thats the confusion part.

Reply:
https://github.com/astronomer/ap-airflow/pull/202/commits/518970a470dcda57119352a3298ec54bac05b1d5 -- I just pushed a commit

Reply:
Thanks